package.json

```diff
@@ -73,7 +73,7 @@
     "jest": "^26.6.3",
     "minimist": "^1.2.6",
     "pdf-to-excel": "^0.3.2",
-    "prettier": "2.2.1",
+    "prettier": "3.0.3",
```

Comment:
Can you add `^` to the version so that `npm install` will only permit minor patch updates and not major patch updates to the prettier package? More on this [here](https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json).